projects/dsp-ui/src/lib/action/components/string-literal-input/string-literal-input.component.spec.ts

```diff
@@ -0,0 +1,316 @@
+import { HarnessLoader } from '@angular/cdk/testing';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { Component, DebugElement, OnInit, ViewChild } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { ReactiveFormsModule } from '@angular/forms';
+import { MatButtonToggleModule } from '@angular/material/button-toggle';
+import { MatButtonHarness } from '@angular/material/button/testing';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatIconModule } from '@angular/material/icon';
+import { MatInputModule } from '@angular/material/input';
+import { MatInputHarness } from '@angular/material/input/testing';
+import { MatMenuModule } from '@angular/material/menu';
+import { MatMenuHarness, MatMenuItemHarness } from '@angular/material/menu/testing';
```

Comment:
the unused imports should be removed

Reply:
That's a good point. @mdelez please use the `shift+alt+o`-command to organize imports in vsc.

Reply:
done in 7276257, pretty nifty command :)